content/release-notes/1.17.0.md

```diff
@@ -0,0 +1,42 @@
+---
+date: 2020-07-13T08:00:00-07:00
+linktitle: "1.17.0"
+title: "1.17.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+* GitOps - Added Github Enterprise support.
+* Snapshots - Added a "Internal Storage (Default)" option to the providers list for KURL embedded clusters. Which will switch back to the default configuration that velero was set up with when the app was installed.
```

Comment:
@austinchambers there's probably a better wording for this. any ideas?

Reply:
```suggestion
* Ability to change to a different snapshot storage provider using the UI on the Snapshot Settings page. 
* "Internal Storage" snapshot provider option has been added to the snapshot providers list for kURL embedded clusters. This allows the user to easily return to the default snapshot configuration when another provider (such as AWS S3 or Google Storage) has been configured. 
```